js/src/data/actions.js

```diff
@@ -559,6 +546,109 @@ export function* updateGoogleMCContactInformation(
 	}
 }
 
+/**
+ * Requests a phone verification code and returns a `verificationId` which is used for the next verification step.
+ *
+ * @param {string} country The country code. Example: 'US'.
+ * @param {string} phoneNumber The phone number string in E.164 format. Example: '+12133734253'.
+ * @param {'SMS'|'PHONE_CALL'} method The verification method.
+ * @return {Object} `{ verificationId: string }`
+ * @throws {Object} `{ display: string }` Will throws an identifiable error with the next step instruction for users.
```

Comment:
:nail_care: a bit nicer VSCode hints
```suggestion
 * @return { { verificationId: string } } Verification id to be used for another call.
 * @throws { { display: string } } Will throws an identifiable error with the next step instruction for users.
```